lib/png12-0.xml

```diff
@@ -8,8 +8,35 @@ PNG (Portable Network Graphics) format files.
 
 This package contains the runtime library files needed to run software
 using libpng.</description>
+  <homepage>http://gnuwin32.sourceforge.net/packages/libpng.htm</homepage>
   <package-implementation package="libpng12-0"/>
   <package-implementation package="libpng12"/>
+  <package-implementation distributions="Gentoo" package="media-libs/libpng"/>
+  <package-implementation package="libpng"/>
+  <group arch="Windows-*" license="zlib/libpng License" >
```

Comment:
Best to use `arch="Windows-i486"` for 32-bit DLLs

Reply:
Done